Sources/FrontEnd/TypeChecking/TypeChecker+ConstraintGeneration.swift

```diff
@@ -740,6 +766,170 @@ extension TypeChecker {
     return facts.constrain(subject, in: program.ast, toHaveType: TupleType(elementTypes))
   }
 
+  // MARK: Patterns
+
+  mutating func infer(
+    typeOf subject: AnyPatternID,
```

Comment:
```suggestion
  mutating func inferType(
    of subject: AnyPatternID,
```

This is basic Swift API guidelines stuff.
Arguably, the label could be dropped.

Reply:
I planned on making the change, but that should be in a separate PR.